invenio_rdm_records/records/jsonschemas/records/record-v2.0.0.json

```diff
@@ -123,7 +123,8 @@
       "additionalProperties": false,
       "required": ["identifier", "provider"],
       "properties": {
-        "identifier": {"$ref": "#/definitions/identifier"},
+        "scheme": {"$ref": "#/definitions/scheme"},
```

Comment:
As per the issue description, the `scheme` would not go here. It would be the key inside the pids object.

```
"pids": {
        "doi": {
            "identifier": "10.5281/zenodo.1234",
            "provider": "datacite",
            "client": "zenodo"
        },
```

we are missing `client` though. What is the benefit of having it as key instead of `scheme`, if its access patterns as dict saves us loops it makes sense. Otherwise, shall we move it inside (as you put it).